balancer/grpclb/grpclb_remote_balancer.go

```diff
@@ -98,11 +99,12 @@ func (lb *lbBalancer) processServerList(l *lbpb.ServerList) {
 	lb.cc.UpdateBalancerState(lb.state, lb.picker)
 }
 
-// refreshSubConns creates/removes SubConns with backendAddrs. It returns a bool
-// indicating whether the backendAddrs are different from the cached
-// backendAddrs (whether any SubConn was newed/removed).
+// refreshSubConns creates/removes SubConns with backendAddrs.
+//
 // Caller must hold lb.mu.
 func (lb *lbBalancer) refreshSubConns(backendAddrs []resolver.Address, fromGRPCLBServer bool) {
```

Comment:
It seems `refreshSubConns` is always followed by `regeneratePicker`. Can we put `regeneratePicker` inside `refreshSubConns`?

Reply:
Done